moped-editor/src/components/GridTable/GridTableFilters.js

```diff
@@ -159,7 +159,10 @@ const GridTableFilters = ({
     return (
       field.lookup_table &&
       !loading &&
-      ["_eq", "_neq"].includes(field.gqlOperator)
+      [
+        "string_equals_case_sensitive",
+        "string_does_not_equal_case_sensitive",
+      ].includes(field.operator)
```

Comment:
This change helps us rely on whichever GraphQL operator is set for the GridTable strings operators set [here](https://github.com/cityofaustin/atd-moped/blob/5b991e54b41d829b6d66291bc8c9e293eba05ed8/moped-editor/src/components/GridTable/GridTableFiltersCommonOperators.js#L152-L165)